README.md

```diff
@@ -76,6 +77,8 @@ $scopes = ['User.Read', 'Mail.ReadWrite'];
 $graphServiceClient = new GraphServiceClient($tokenRequestContext, $scopes);
 ```
 
+To initialise the `GraphServiceClient` with an already acquired access token, see section on [access token management](docs/Examples.md#access-token-management).
```

Comment:
Should we also have a section on an hybrid, where it starts without a token and then reuses / refreshes the token for the subsequent calls? This feels more like something that would be useful.

Reply:
Both scenarios are covered in the Access token management section. I've updated this line to reflect that both scenarios are covered in more detail in the examples.